.github/workflows/generate-upload-manifest.yml

```diff
@@ -0,0 +1,110 @@
+name: generate-upload-manifest
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  generate-manifest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@main
+      - name: install jq
+        run: sudo apt-get install jq
+      - name: Make manifest
+        run: |
+          DOWNLOAD_URL_PREFIX="https://raw.githubusercontent.com/sensible-hq/sensible-configuration-library/main/" 
```

Comment:
I'd like this script in a separate file, it's a bit harder to read this way since it doesn't get highlighted